XenModel/Actions/Network/SetSecondaryManagementPurposeAction.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using XenAdmin.Network;
+using XenAPI;
+
+namespace XenAdmin.Actions
+{
+    public class SetSecondaryManagementPurposeAction : AsyncAction
+    {
+        private Pool pool;
+        private List<PIF> pifs;
+        private static readonly log4net.ILog log = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+
+        public SetSecondaryManagementPurposeAction(IXenConnection connection, Pool pool, List<PIF> pifs)

```

Comment:
Any RBAC considerations for this action? Renaming does not sound particularly risky as an operation, but perhaps it should be subject to the same restrictions as the other re-configuration operations?

Reply:
Actually in this action, only PIF.SaveChanges calling to server side is made. Is it enough to add this one to the RBAC precheck?

Reply:
TYes, ok, the actual API call used is pif.set_other_config so let's add this one.